src/styles/blocks/function.scss

```diff
@@ -0,0 +1,17 @@
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
+
+@each $day, $shift in $week {
+  .calendar--start-day-#{$day} .calendar__day:first-child {
+    margin-left: $shift + ($width-cell * $shift);
```

Comment:
The variable `$width-cell` is used but not defined in this file. Ensure it is defined or imported from another SCSS file.